src/components/profile/UserProfile.js

```diff
@@ -26,6 +26,14 @@ class UserProfile extends Component {
         this.toggleExpansion = this.toggleExpansion.bind(this);
     }
 
+    // count the occurence of each quality for later 
```

Comment:
On line 60 the qualities are being mapped. The actual mockData provided can still be parsed to extract the top n qualities using the code below. I know this wasn't a part of this PR but it is relevant and can be fixed here. With this, [STUD-149](https://team-1610936365889.atlassian.net/jira/software/projects/STUD/boards/4?selectedIssue=STUD-149) can also be closed.

```
let allTags = userData.reviews
	.flatMap(obj => obj.tags)
	.reduce((dict, obj) => {
		dict[obj.name] = (dict[obj.name] || 0) + 1;
		return dict
	}, {})

let topFiveTags = Object.keys(allTags)
	.map(key => [key, allTags[key]])
	.sort((x, y) => y[1]-x[1])
        .slice(0, 5)
```